custom/covid/management/commands/update_cases.py

```diff
@@ -42,6 +42,15 @@ def handle(self, domain, case_type, **options):
         else:
             user_id = SYSTEM_USER_ID
 
+        try:
+            if options["location"]:
```

Comment:
this can be rewritten as `options.get('location')` and then you don't need to catch the KeyError. It could even be
`location = options.get('location')` if you remove the warning, then it would be condensed into one line